test/test_p4est_2d.jl

```diff
@@ -296,6 +296,10 @@ end
         t = sol.t[end]
         u_ode = sol.u[end]
         du_ode = similar(u_ode)
+        # Larger for values for allowed allocations due to usage of custom 
```

Comment:
I'm confused by the two `for`s :sweat_smile:

Reply:
Thanks!